---
title: "An update and reassessment of fern and lycophyte diversity data in the Japanese Archipelago"
output: word_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE, include = TRUE, eval = TRUE, cache = TRUE, results = "hide")

# Load packages not used in main plan
library(glue)

# Load targets made from plan
loadd(all_cells)
loadd(cells_per_species)
loadd(cps_by_ploidy_means)
loadd(cps_by_repro_means)
loadd(cps_by_repro_model_summary)
loadd(japan_pterido_tree)
loadd(lat_by_repro_means)
loadd(lat_by_repro_tukey)
loadd(occ_data_pteridos)
loadd(pd_ferns)
loadd(pd_pteridos)
loadd(ppgi)
loadd(repro_data)
loadd(repro_data)
loadd(species_per_cell)
loadd(taxonomic_data)

# Function for rounding with trailing zeros
round_t <- function (x, digits) round(x, digits) %>% sprintf(glue::glue("%.{digits}f"), .)
```

An update and reassessment of fern and lycophyte diversity data in the Japanese Archipelago

Atsushi Ebihara & Joel H. Nitta

## Introduction

Ferns and lycophytes, also traditionally known as ‘pteridophytes’, have received strong attention as biological research material. Although there are only estimated to be 11,916 extant species of ferns and lycophytes (Pteridophyte Phylogeny Group, 2016)---much less than the number of seed plant species (approximately one twelfth)---the number of studies on ferns and lycophytes, especially in the field of systematics and evolution, is relatively large. Thus, ferns and lycophytes are relatively information-rich organisms, especially for biodiversity. However, examples of analyses of fern diversity patterns at the regional scale are scarce (e.g. Bogonovich et al. 2014, Mountier et al. 2018). In the last century, the diversity of ferns and lycophytes in Japan was quite thoroughly studied, and is probably the third best studied area after Europe and North America. Guo et al. (2003) combined trait and distrubtion data to analyze determinants of species diversity, abundance, and distribution of the pteridophyte flora of Japan. The sole serious problem of the analyses of Guo et al. (2003) was the lack of phylogenetic information at that time, but DNA sequence data for nearly all ferns and lycophytes of Japan (at least for the cholorplast) have been produced by subsequent studies (Ebihara 2011, Ebihara et al. 2010). In addition, it should be noted that the occurrence data based on Kurata and Nakaike (1979, 1981, 1983, 1985, 1987, 1990, 1994, 1997) used by Guo et al. (2003) did not cover all known species at the time, and more have been revised or added since. <!-- Check that Guo et al. didn't include all species *at the time*-->
Ebihara (2016, 2017) provided updated distribution maps for all the native taxa of ferns and lycophytes in Japan reflecting numerous new occurrences and recircumscription of taxa based on the results of recent studies, but did not make the data available online. In the current study, we make available the most recent diversity datasets on ferns and lycophytes in Japan, review recent progress and outstanding questions in documenting this flora, and reassess the results of Guo et al. (2003). 

## Flora

```{r flora-numbers}
n_taxa <- taxonomic_data %>% pull(taxon_name) %>% n_distinct
n_species <- taxonomic_data %>% pull(species_name) %>% n_distinct
n_genera <- taxonomic_data %>% pull(genus) %>% n_distinct
n_families <- taxonomic_data %>% pull(family) %>% n_distinct

# Make sure the number of ppgi families in our data match
# the official number
assert_that(ppgi %>% pull(family) %>% n_distinct == 51)
```

Study on the fern and lycophyte flora of the Japanese Archipelago began with 44 species enumerated in Thunberg (1786)’s “Flora Japonica”. Thereafter the number of recognized native taxa has been gradually increasing: 460 species including subspecies and varieties (Makino and Nemoto 1925), ca. 400 species excluding those in Ogasawara (Bonin) and Ryukyu Islands (Tagawa 1959), and 630 species (Iwatsuki 1995). The latest checklist of native taxa of Japan, which we use in this study, is the ‘FernGreenList ver. 1.01’ (in Japanese; Ebihara et al. 2017a). It comprises `r n_species` species (`r n_taxa` taxa including subspecies and varieties; hereafter called ‘taxa’) belonging to `r n_families` of the 51 families accepted in PPGI (Pteridophyte Phylogeny Group 2016) [Electronic supplementary material 1]. As almost no uninvestigated area remains in the archipelago, most of the new additions to the flora after the 1990s have been newly segregated species from previously known ones based on evidence from ploidy levels, enzyme analyses and/or DNA sequences (e.g., Ebihara et al. 2019a, Fujiwara et al. 2018, Hori et al. 2015, 2018a, 2018c, Lin et al. 1994, 1996, Masuyama and Watano 2010, Murakami et al. 1999, Ohta and Takamiya 1999, Serizawa 2015, Takamiya et al. 1997) along with only a small number of new records (Ebihara et al. 2014, 2019a). The number of taxa in Japan is nearly identical to that in Taiwan (721 taxa excluding forms and hybrids, Knapp 2014). The area of strongest floristic affinity is continental China, with which Japan shares 487 taxa (67.5%), almost corresponding to two-thirds of the native taxa in Japan (Ebihara 2016, 2017). The most recent information as of writing identified 125 endemic taxa (17.3% of the native taxa) to Japan (Ebihara 2018). 

## Interspecific hybrids

The huge number of interspecific natural hybrid taxa (371 combinations listed in Ebihara 2017) known in addition to the 721 native, non-hybrid taxa is an outstanding characteristic of the Japanese pteridophyte flora. This is much more than the number of known hybrid combinations in North America (xxx), China (10 were formerly accepted in Flora of China [Wu et al. 2013], and 20 or more combinations were suggested in the notes), or Taiwan (13 listed in Knapp 2014). Such high diversity is most likely a result of efforts on seeking new hybrid combinations by Satoru Kurata (1922-1978) and his pupils, but might be, at least in part, a geographical effect of the long north-south axis of the Japanese archipelago, which may enable sympatric occurrence of species with different habitat preferences concomitant with climate fluctuation. It is highly likely that most of the interspecific hybrid individuals in a large part of the world are yet unrecognized, including cases of misidentification as non-hybrid taxa. It should be noted that only a few combinations of hybridity in Japan have been verified by artificial crossing, enzyme patterns, or nuclear DNA markers; the others are putative hybrids based on morphology. An expedient method to identify F1 hybrids is to observe the size and shape of spores; ill-formed or aborted spores are often due to failure of chromosomes from divergent species to properly pair at meiosis. However, this is not always reliable in the case of homoploid hybrids, which often have normal-shaped, regular spores, e.g., *Botrychium* x *silvicola* (Ebihara et al. 2015, Sahashi 1983) and other putative homoploid hybrids of *Botrychium* species in Japan (N. Sahashi, personal communication). An incomplete establishment of reproductive isolation was observed between *Osmunda lancea*, a rheophytic species endemic to Japan, and its closest relative *O. japonica* (Yatabe et al. 2009, 2011). Their ‘hybrid’ *O.* x *intermedia* actually consists of hybrid swarms including not only F1 but also F2 or later-generation hybrids. Fern interspecific hybrids have long been believed to occur in places where sporophytes of their two parent species are sympatric, and this idea was an important grounds for identification of putative parentages. However, our studies demonstrated a number of examples of sterile interspecific hybrids unaccompanied by their parental species. One of the extreme cases is found in the dennstaedtioid fern *Monachosorum* x *arakii* which was originally considered as an endemic species to Japan. Ebihara et al. (2016) concluded that it is a sterile hybrid between *Monachosorum henryi* and *M. nipponica*. The former is currently distributed in Taiwan, continental China, and Himalaya, but not Japan. Although *M.* x *arakii*  is a somewhat special case with strong vegetative reproduction ability---it can asexually reproduce from both rhizomes and frond-borne bulbils---making its relict distribution possible, we should keep in mind that current species distributions may not restrict the possible combinations of putative parents when constructing hypotheses of hybrid origins. Apart from relict distribution, there is also strong evidence for the contribution of independent gametophytes to formation of hybrid sporophytes in *Vandenboschia* in Japan (Ebihara et al. 2009).

## Occurrence data and regional species richness

```{r occ-data-numbers}
# Number of 10km2 grid cells total across Japan
n_grid_cells <- all_cells %>% pull(secondary_grid_code) %>% n_distinct
# Number of cells with at least one species observed
n_cells_with_occ <- occ_data_pteridos %>% pull(secondary_grid_code) %>% n_distinct

# What is the most common species (occuring in the greatest number of grid cells)?
most_common_sp <- cells_per_species %>% arrange(desc(n)) %>% slice(1) %>% pull(taxon_name)
# How many grid cells does it occur in?
most_common_sp_count <- cells_per_species %>% arrange(desc(n)) %>% slice(1) %>% pull(n)
# Verify that the most common sp is what we mention in the text
assert_that(most_common_sp == "Pteridium aquilinum subsp. japonicum")

# What is grid cell with the most species?
highest_richness_cell <- species_per_cell %>% arrange(desc(n)) %>% slice(1) %>% pull(secondary_grid_code)
# How many taxa does it have, excluding hybrids?
highest_richness_cell_n_taxa <-
occ_data_pteridos %>%
  filter(secondary_grid_code == highest_richness_cell) %>%
  filter(taxon_id %in% repro_data$taxon_id) %>%
  pull(taxon_id) %>%
  n_distinct

# What is grid cell with the second-most species?
highest_richness_cell_2 <- species_per_cell %>% arrange(desc(n)) %>% slice(2) %>% pull(secondary_grid_code)
# How many taxa does it have, excluding hybrids?
highest_richness_cell_n_taxa_2 <-
occ_data_pteridos %>%
  filter(secondary_grid_code == highest_richness_cell_2) %>%
  filter(taxon_id %in% repro_data$taxon_id) %>%
  pull(taxon_id) %>%
  n_distinct

# What is grid cell with the third-most species?
highest_richness_cell_3 <- species_per_cell %>% arrange(desc(n)) %>% slice(3) %>% pull(secondary_grid_code)
# How many taxa does it have, excluding hybrids?
highest_richness_cell_n_taxa_3 <-
occ_data_pteridos %>%
  filter(secondary_grid_code == highest_richness_cell_3) %>%
  filter(taxon_id %in% repro_data$taxon_id) %>%
  pull(taxon_id) %>%
  n_distinct

```

The eight volumes of "Illustrations of Pteridophytes of Japan" (Kurata and Nakaike, 1979, 1981, 1983, 1985, 1987, 1990, 1994, 1997) were a remarkable achievement of citizen-scientists to document species distributions at high resolution throughout an entire country. In addition to a brief description of each species, these books featured 10 km^2^ grid-cell distribution maps based on vouchers of ca. 180,000 specimens collected by the members of the Nippon Fernist Club (NFC). Surveys of the NFC have continued to the present, and a large part of the specimens presently deposited in the herbarium of National Museum of Nature and Science (TNS) were recently re-identified accommodating up-to-date species circumscriptions and new collections by club members, resulting in an updated set of 10 km^2^ grid-cell maps including ca. 35,000 new grid-cell records (Ebihara 2016, 2017). Occurrences based on specimens are usually regarded as presence-only data, but the distribution maps of Ebihara (2016, 2017) can be virtually regarded as presence/absence data due to the extensive collection effort of NFC members. For the convenience in accessing and reusing the data, we make the data used to construct the distribution maps in Ebihara (2016, 2017) available here as a spreadsheet of species occurrences (presence-only) per grid cell (`r nrow(occ_data_pteridos) %>% comma` rows; Electronic supplementary material 2). Our policy for the dataset is to only include occurrences based on specimens or specimen-images examined by ourselves (the only exception is *Isoetes* occurrences, which are cited from a specimens list provided by Takamiya et al. [1997]). At least one species is recorded in `r n_cells_with_occ` out of `r n_grid_cells` grid cells total (`r percent(n_cells_with_occ/n_grid_cells)`), and the most abundant species is *Pteridium aquilinum* subsp. *japonicum* (Dennstaedtiaceae), which was recorded in `r most_common_sp_count` grids (`r percent(most_common_sp_count/n_grid_cells)`).

Although not included in the present dataset, a much larger number of specimen records without images are available on-line via Science Museum Net, the local data portal for museum collections in Japan  (http://science-net.kahaku.go.jp/), and the Global Biodiversity Information Facility (GBIF) data portal (https://www.gbif.org/). As of May 23, 2019, Japan is the country with the largest number of fern and lycophyte specimens in GBIF (531,311 occurrences).

Species richness per 10 km^2^ grid-cell excluding nothotaxa is shown in Fig. 1a. The grid cells with the highest (`r highest_richness_cell_n_taxa` taxa, grid code: `r highest_richness_cell`) and the second highest  (`r highest_richness_cell_n_taxa_2` taxa, grid code: `r highest_richness_cell_2`) richness are found in the mountainous area of Yakushima Island, located south of Kyushu. The third highest grid (`r highest_richness_cell_n_taxa_3` taxa, grid code `r highest_richness_cell_3`) is located in the area covering Nachi-no-taki Falls in southern part of Wakayama Prefecture, southern Honshu.

```{r pd-analysis-numbers}
pd_ferns <- arrange(pd_ferns, desc(pd_obs))
pd_pteridos <- arrange(pd_pteridos, desc(pd_obs))
```

Species richness is only one dimension of biodiversity; phylogenetic diversity is also important to consider for understanding biogeographic patterns and setting conservation priorities (CITE). We updated the *rbcL* sequence dataset of Ebihara et al. (2010) by making several additions and corrections, which resulted in coverage of `r percent(Ntip(japan_pterido_tree)/nrow(repro_data))` of the native taxa (Electronic supplementary material 1; newly generated sequences deposited in GenBank). We carried out standard phylogenetic analysis by aligning the sequences with MAFFT (CITE) and inferring the phylogenetic tree with MrBayes (CITE). We then calculated Phylogenetic Diversity (PD; Faith 1992, 2006) for each 10 km^2^ grid-cell by trimming the tree to only taxa present in the cell and summing the branchlengths (excluding the root). We carried out analyses on a dataset including all ferns and lycophyte taxa (Fig. 1b) and a dataset including ferns only (REF FIG), since the deep divergence between ferns and lycophytes may bias measures of PD when lycophytes are included. The highest PD value was obtained in a grid in Wakayama Prefecture (grid code: `r pd_pteridos$site[1]`), the second highest in the northern part of the Okinawa Island (grid code: `r pd_pteridos$site[2]`) and third highest in Iriomote Island (grid code: `r pd_pteridos$site[3]`), the latter two located in the southwestern-most part of the archipelago. In the result of PD analyses, the Yakushima grid (grid code: 453034) with maximum species richness was ranked `r which(pd_pteridos$site == "453034")`th in the PD analysis using ferns+lycophytes, and `r which(pd_ferns$site == "453034")`th in that only using ferns, respectively. The disparity of the results might suggest the difference in their constituent species---species richness in Yakushima Island is a result of the presence of some species-rich genera (e.g., 19 taxa of *Dryopteris*, 16 taxa of *Diplazium*, 14 taxa of *Athyrium*).

```{r sex-numbers}
# Format means for printing inline
# - Cells per species by reproductive mode
cps_by_repro_means_inline <- 
  cps_by_repro_means %>%
  rename(repro = reproductive_mode) %>%
  mutate_at(vars(mean, sd), ~round_t(., 1)) %>%
  mutate(
    stat = glue("{mean} ± {sd} (*n* = {n})")
  )

# - Cells per species by ploidy level of sexual taxa
cps_by_ploidy_means_inline <- 
  cps_by_ploidy_means %>%
  mutate_at(vars(mean, sd), ~round_t(., 1)) %>%
  mutate(
    stat = glue("{mean} ± {sd} (*n* = {n})")
  )

# - Latitudinal range per species by reproductive mode
lat_by_repro_means_inline <-
  lat_by_repro_means %>%
  rename(repro = reproductive_mode) %>%
  mutate_at(vars(mean, sd), ~round_t(., 2)) %>%
  mutate(
    stat = glue("{mean} ± {sd} (*n* = {n})")
  )

```

Our updated occurrence data support the trend that species with seasonal-green phenology have wider distribution than evergreen species as suggested by Guo et al. (2003), but we believe this is probably an artifact caused by small land areas at lower altitudes in the country where evergreen species dominate.
<!-- Clarify what Ebihara means by previous, specifically "country" -->
Guo et al. (2003) also suggested that species with multiple reproductive modes are more widely distributed than species with a single reproductive mode. However, our data did not support this trend (mean numbers of grids, sex.+apo. = `r cps_by_repro_means_inline %>% filter(repro == "sex_apo") %>% pull(stat)`, sex. = `r cps_by_repro_means_inline %>% filter(repro == "sexual") %>% pull(stat)`, apo. = `r cps_by_repro_means_inline %>% filter(repro == "apomictic") %>% pull(stat)`; analysis of variance, *p* = `r cps_by_repro_model_summary %>% filter(term == "reproductive_mode") %>% pull(p.value) %>% round_t(2)`). The result of Guo et al. (2003) seems to be strongly affected by their species circumscription, and presumably some “species with multiple reproductive modes” in are separated into multiple taxa each with a single reproductive mode in the present treatment. Even though they are not significantly different in the number of grid cells they occupy, apomictic taxa show significantly narrower latitudinal ranges than sexual ones (mean latitudinal degree of ranges: sex. = `r lat_by_repro_means_inline %>% filter(repro == "sexual") %>% pull(stat)` and apo. `r lat_by_repro_means_inline %>% filter(repro == "apomictic") %>% pull(stat)`, *p* = `r lat_by_repro_tukey %>% filter(comparison == "sexual-apomictic") %>% pull(adj.p.value) %>% round_t(3)`; Tukey's HSD). This coincides with the result of Tanaka et al. (2014) that found smaller range sizes of apomicitic taxa when measured by both latitude and elevation. We also observed that the number of apomictic species tends to increase towards the south (Fig. 1d, e), in agreement with Tanaka et al. (2014), who attributed this pattern to the poor cold-hardiness of gametophytes of apomictic species due to their larger cell sizes. The trend of narrower distribution of polyploid taxa (Guo et al. 2003) was not supported by present data excluding apomictic species (average number of grids, sexual diploid = `r cps_by_ploidy_means_inline %>% filter(sexual_diploid == TRUE) %>% pull(stat)`, sexual polyploid = `r cps_by_ploidy_means_inline %>% filter(sexual_polyploid == TRUE) %>% pull(stat)`). It is likely that the trend reported by Guo et al. (2003) was probably due to the narrower distribution of apomictic species, which are dominated by triploids.

## Cytotaxonomic data

Even today, chromosome numbers of ferns and lycophytes which are important clues for knowing ploidy level and reproductive mode, and were previously used to infer phylogenetic affinities before the introduction of DNA sequencing. Researchers in Japan, stimulated by Irene Manton’s leading work “Problems of Cytology and Evolution in the Pteridophyta” (Manton 1950), actively made chromosome counts of ferns and lycophytes using the squash technique in the second half of the 20th century. More than 150 papers on chromosome numbers of Japanese ferns and lycophytes have been published, and all the counts on Japanese material before 1996 were indexed by Takamiya (1996). Additions have been continuously made, and Nakato and Ebihara (2016) summarized that the information drawn from Japanese individuals is available for 74 % of the native taxa (61 %, if nothotaxa are counted). This is a much higher level of coverage compared with other areas of East Asia (e.g., ca. 27 % in China, Cheng and Zhang [2010]), and the second highest coverage in Asia following India (ca. 89 %, Bir and Verma [2010]). Further addition is sorely needed for ascertaining some early records that lack voucher specimens, and for detecting infraspecific polyploidy and/or multiple reproductive modes overlooked by previous superficial sampling, as well as unsampled taxa. Flow cytometry has recently been used for inferring ploidy levels of large number of samples belonging to closely related species groups (e.g., Ebihara et al. 2005, Fujiwara et al. 2018, Hori et al. 2015, Nitta et al. 2011), but the method always requires at least one control sample whose ploidy level was confirmed by chromosome counts. Flow cytometry analyses using not only fresh leaves but also spores from recently collected herbarium specimens might accelerate data accumulation (Kuo and Huang 2017).  

## Threatened species

Latest national red list of Japan (Ministry of the Environment, Japan 2019) included more than one third of the native fern and lycophyte taxa (260 in total, 6 EX, 3 EW, 85 CR, 59 EN, 68 VU, 38 NT and 1 DD; see Fig. 1f for the distribution of richness). The largest threat for ferns is herbivory of the sika deer distributed throughout the archipelago which obviously increased after 1980’s (Takatsuki 2009). In particular, species growing on the forest floor (e.g. Athyrium, Terada and Takamiya 2006) undergo severe damage, and several narrowly distributed species in Kyushu and Yakushima Island became almost extinct or only found inside deer fences. Although national law for endangered species is recently expanding objective fern and lycophyte species which are proscribed collection and sellout (28 species as of May 2019), it is apparently not directly effective for preventing herbivory of deer. There are still a few reports of conservation activities and researches on ferns and lycophytes in Japan: ubiquitous genotyping of Athyrium viridescentipes (Izuno et al. 2012) and clarifying origin and propagation from spore in Dryopteris shibipedis (Ebihara et al. 2012) which was extinct in the wild. Despite 70 of the 260 taxa in the latest national red list are endemic species to Japan, they are neither assessed nor listed in the IUCN Red List as globally threatened species as of May 2019 (IUCN 2019).

## Introduced species

Compared with flowering plants, number of naturalized species of ferns and lycophytes is quite small in the Japanese Archipelago, and they are almost uninvolved with the invasive species issue. Selaginella uncinata (Selaginellaceae) and Pityrogramma calomelanos (Pteridaceae) are two of scarce examples of relatively widespread introduced species in the country (Ebihara 2016). Dryopteris intermedia (ref), D. carthusiana (ref) (Dryopteridaceae) and Dennstaedtia punctilobula (Dennstaedtiaceae) (Ueno et al. 2019) were probably introduced with sprayed seeds on banks and only collected in each one or a few localities. Azolla cristata (Salviniaceae) is the only invasive fern species controlled by national low (Invasive Alien Species Act), but is often troubled in identification due to morphologically similarlity to an endangered and endemic species, A. japonica. Another similar species, Azolla filiculoides is still controversial on its natural distribution in Japan (Ebihara 2016, Suzuki et al. 2005). Selaginella moellendolffii (Selaginellaceae), Psilotum nudum (Psilotaceae), Adiantum capillis-veneris, Pteris vittata (Pteridaceae) and Thelypteris dentata (Thelypteridaceae) have certain natural distribution in the subtropical areas of the country, but are more frequently found as escaped plants from greenhouses. Murakami et al. (2008) demonstrated a range expansion of Thelypteris dentata facilitated by urbanization in Kinki District, Japan. The natural habitat of Cyrtomium falcatum subsp. falcatum is restricted to coastal areas (Ebihara et al. 2017a, Matsumoto 2003), and scattered occurrences in urban environments (Ebihara 2017) in inlands are probably introduced ones.

## Reticulate evolution

Species complexes or aggregate species exhibiting wide ranges of continuous morphological variation are quite commonly known in ferns and lycophytes (Haufler 1996), and this trend seems to be true for the flora of Japan. In case of that interspecific F1 hybrids make the species boundaries ambiguous, we could usually find distinct gaps between them by removing their hybrid individuals suggested by irregular-shaped spores with low germination rates (e.g. Ohta and Takamiya 1999 in the Diplazium mettenianum complex). When interspecific hybrids overcome sterility by chromosome doubling, the situation becomes more complicated and the biological units composing the complex are not easily clarifiable solely by morphological approach. The allopolyploids of hybrid origins often re-hybridize with their parental species --- such "reticulate evolution" process became well-known after the study on Appalachain Asplenium (Wagner 1954). Apparently similar approach combining chromosome and morphological observations was rarely adopted to Japanese species. After 1990's, allozyme analyses were frequently applied to the fern complexes in Japan, and plastid sequences often accompanied as indicators of maternal lineage (e.g. Darnaedi et al. 1990 in Dryopteris yakusilvicola and its related species, Lin et al. 1996 in Odontosoria, Shinohara et al. 2010 in Lepisorus, Takamiya and Ohta 2001 in Diplazium). In 2000's, single or low copy nuclear DNA regions became popularly used as biparently inherited markers (e.g. Ishikawa et al. 2002), and sequence polymorphisms were efficiently isolated by cloning or the SSCP (Single Strand Conformation Polymorphism) method (initiated by Ebihara et al. 2005 in the Vandenboschia radicans complex, thereafter applied to various fern groups including Ceratopteris [Adjie et al. 2007]), the Pteris cretica complex [Jaruwattanaphan et al. 2013], the Davallia repens complex [Chen et al. 2014], the Asplenium normale complex [Fujiwara et al. 2017]). Although next-generation sequencing is widely used in phylogenetic and population genetic studies on various organisms in recent years, it is still not popularly applied to analyses on species complexes of ferns and lycophytes mainly due to technical difficulties in removing chimeral contigs identifying and homologous copies from the obtained sequences. More recently Rothfels et al. (2017) succeeded in phylogenetic analysis on a polyploid complex of cystoptreroid ferns using PacBio sequences, and the method is highly expected to apply wide ranges of fern complexes.

## Species complexes involved with apomixis

```{r apomixis-numbers}
percent_apo <-
count(repro_data, reproductive_mode) %>%
  mutate(
    total = sum(n),
    percent = n / total
  ) %>%
  filter(reproductive_mode == "apomictic") %>%
  pull(percent) %>%
  round(2)
  

```

Further complexity is provided when apomictic reproduction (reviewed in Grusz et al. 2016) is concerned. The topic, apomixis or apogamy in ferns has attracted researchers in Japan as the reproductive mode is known in a higher rate in the native taxa of the country (13 % of the species whose reproductive modes are known in Takamiya 1996; `r percent(percent_apo, suffix = " %")` in present study) than the average rate in the world (ca. 10 %, Grusz et al. 2016). Apomixis plays a role not only in overcoming sterility of interspecific hybrids (e.g. Park and Kato 2003) but also making secondary hybridization by fertilization between a sperm produced by the ‘hybrid’ and an egg of sexually reproducing races. This process can generates countless apomict races with various genomic combinations which are morphologically continuous, and the races are often controversial about circumscription of ‘species’ (a polymorphic species versus numerous morphologically ill-defined species). One of the best known apomict complexes troublesome in identification, the Dryopteris varia complex was recently clarified their internal relationships using single-copy nuclear DNA markers (Hori et al. 2014, 2015, 2016), and analyses on several other complexes are in progress. However, even using molecular markers, we often meet an obstacle that we cannot find progenitors or sexual diploids of the complex. Hori et al. (2018d) discovered several sexual diploid individuals of Dryopteris in southern China, and as the progenitors the newly found diploids well explained the reticulate evolution processes of the D. erythrosora complex in Japan which had bothered researchers for many years (Hori et al. 2018b). This case supported the hypothesis by Gao et al (2003): “The ancestral sexual species of many Japanese apogamous species occur in China, suggesting that these species might have migrated from China”. In contrast to these successful clarifications, we are still struggling with more challenging complexes. The species of Cyrtomium (Dryopteridaceae) in Japan typically match this pattern: 12 native species and subspecies are all triploid apomicts except for two subspecies of C. falcatum (subsp. australe and subsp. littorale, sexual diploid) and C. devexiscapulae (sexual tetraploid) (Ebihara et al. 2017a, Matsumoto 2003). Despite careful exploration of sexually reproducing individuals by examining spore number per sporangium in herbarium specimens collected in China, a large part of them are apomictic and sexual ones were found less frequently in geographically narrow ranges (Mitsuta 1986, 1988). The Pteris fauriei complex is probably in similar situation, and it includes several endemic ‘species’ to Japan which are diploid or triploid apomict (Ebihara 2016). The “Diploids-First Approach” (Beck et al. 2010) is a principle for the studies on fern reticulate complexes, but we should mind that diploids are not always present nearby including possible extinction in complexes involved with apomixis.

## Species concept

```{r species-concept-numbers}
n_sex_polyploid <-
repro_data %>%
  filter(sexual_polyploid == TRUE & reproductive_mode == "sexual") %>%
  nrow
```

It is obvious that recent increase of recoded taxa in Japan is largely owing to transition from morphological species concept to biological or evolutional ones. In many cases, subsequent taxonomic revisions were published after analyses on certain species complexes. Although we have not reached any final consensus on the concept, strict application of neither biological species concept nor evolutionary species one seem to be preferred by recent studies. In particular, an autotetraploid, which should be treated as an independent species from a diploid sharing same genome in the biological species concept, is more often treated as infraspecific polyploidy. Ebihara et al. (2005, 2009) circumscribed a new allotetraploid species Vandenboschia hokurikuensis including recurrently originated (at least twice, based on maternally inherited plastid sequences). In practice, every allopolyploid with different combination of diploid progenitor species requires a name. Among the `r n_sex_polyploid` taxa in Japan which sexual polyploid (tetraploid or higher) was reported for, evidences for judging autopolyploidy/allopolyploidy are available in 34 taxa (`r (34 / n_sex_polyploid) %>% percent(suffix = " %")`). Only four of them (Asplenium boreale [Fujiwara et al. 2017], Asplenium pekinense [Lin et al. 2013], Pteris deltodon [Jaruwattanaphan et al. 2013], and Lepisorus tosaensis [Fujiwara et al. 2018]) are probable autopolyploid (autotetraploid) and the remaining 30 taxa are allopolyploid. Some of currently accepted infraspecific taxa defined to correspond to ploidy levels (e.g. diploid Lycopodium clavatum var. nipponicum vs. tetraploid var. asiaticum [Takamiya 1989]; tetraploid Deparia petersenii var. petersenii vs. hexaploid var. yakusimensis [Shinohara et al. 2003]) might be classified into different species if future studies clarify differences in their genomic combinations.

We are often faced with a problem about taxonomic treatment for apomictic races. As they are asexually reproducing, the biological species concept is not applicable, and in fact several studies had no other choice than suspension of corresponding taxonomic treatment (e.g. Chen et al. 2014, Nitta et al. 2011). Another trouble is on their nomenclatural categories: i.e. should we describe apomictic races of hybrid origin as nothotaxa? Recircumscription of species recently made by Hori et al. (2018a) for the Dryopteris varia complex is probably the most easily acceptable decision, which is, providing a species name at non-hybrid category for every apomictic race with different combination of sexually reproducing progenitors. This treatment resulted a split of Dryopteris hikonensis to three apomictic species namely D. hikonensis s.s. (= D. protobissetiana x D. varia), D. erythrovaria (= D. caudipinna x D. protobissetiana x D. varia) and D. subhikonensis (= D. protobissetiana x D. saxifraga x D. varia). It should be noted that we treated some individuals originated from hybridization between an apomictic race and sexual species which showing lower spore germination rates and/or growing not vigorously in the wild as nothotaxa in FernGreenList (Ebihara et al. 2016). The categorization largely depends on our observation and experiences, and we could not always strictly border them.
 There is no standard taxonomic treatment for the case that both sexual race and apomictic race coexist in a single species. While in many cases both reproductive modes are not distinguished taxonomically (e.g. Pteris oshimensis, Phegopteris connectilis and Dryopteris chinensis), the apomicitc race of Pteris terminalis are distinguished at varietal rank (var. fauriei, Ebihara et al. 2017b) considering its morphological distinctness and failure of finding unique allele in the apomictic race.

## Gametophytes

Our knowledge about gametophytes of ferns and lycophytes is considerably limited compared with that in sporophytes, nevertheless Japan is one of the world’s most information-rich areas for fern gametophytes. Two dennstaedtioid genera, Coptodipteris (‘Coptidipteris’) and Fuziifilix described by Nakai and Momose (1937) are scarce examples segregated based on gametophyte characters, even though they are probably merged into one of the two clades of Dennstaedtia (Perrie et al. 2015). Momose (1967) observed and illustrated gametophytes of ca. 330 species mostly native to the country using cultured ones from spores. Although he succeeded in culture almost only for cordate gametophytes, subsequent researchers observed non-cordate gametophytes which grow and maturate usually slower than cordate ones (e.g. Hymenophyllaceae [Yoroi 1972, 1976, 1992], Haplopteris (Pteridaceae) [Yoroi 1975], Pleurosoriopsis (Polypodiaceae) [Ebihara et al. 2019b, Masuyama 1975] and Botrychium (Ophioglossaceae) [Takahashi and Imaichi 2007]).

Ebihara et al. (2010) completed setting up a country scale DNA barcode library (plastid rbcL and trnH-psbA regions, covering ca. 94 % of native fern and lycophyte taxa) which is quite useful for identification of gametophytes in the wild. Our preliminary sampling demonstrated an interesting phenomenon that non-cordate gametophytes frequently ‘independent’ or occur far from their counterpart sporophytes localities while cordate ones are almost always sympatric with conspecific sporophytes (Ebihara et al. 2013). This is also true for independent filamentous gametophytes of Vandenboschia as mentioned above, and some of the independent gametophytes might be solely gametophytes distributed in the country (Haplopteris sp. in Yakushima Island, Kuo et al. 2017). Expanded samplings of gametophytes in the wild will shed light on more and more independent ones. In another aspect, identification using DNA barcode enabled detection of fungal association of wild fern gametophytes (Ogura-Tsujita et al. 2013, 2016). Although they discussed possible correlation between gametophyte cushion structure and arbuscular mycorrhizal association, much wider taxonomic and geographical samplings are necessary for understanding the evolutional trend as well as for comparison with symbiotic fungi in sporophytes (reviewed by Lenhert et al. 2016, Pressel et al. 2016).

## Taxonomic key characters

Ebihara (2016, 2017) publicized matrices for major morphological characters frequently used for identification keys for all the 721 native ferns and lycophytes in Japan. All the quantitative characters were newly measured by the members of the Nippon Fernist Club in ten randomly selected herbarium specimens per taxon. As these data are much more useful in an electric form than a printed version, we provide the data file (Electronic supplementary material 3) in the format for Lucid 3.3, which is a free version of the software for interactive keys available at http://www.lucidcentral.com/en-us/home.aspx.

## Future perspective

Albeit none of the genus accepted in PPGI (Pteridophyte Phylogeny Group 2016) is endemic to the archipelago, recent studies revealed several infrageneric clades distributed centered in and/or representing the highest diversity there. Spicantopsis, the most recently resurrected genus in Blechnaceae is now subendemic to Japan (2 endemic species and 1 species in Japan and Taiwan) and its ancestor estimated to migrate from North America to East Asia in ca. 85 mya (Gabriel y Galan et al., in press). Apart from Spicantopsis, we could find at least three groups exhibiting significant diversity in there: Deparia subsect. Athyriopsis (Kuo et al. 2016, 2018), “Clades VI + VII” of Lepisorus (Wang et al. 2009), “the HYSUFI clade” of Polystichum (Le Pechon et al. 2016). Clarification of their speciation and biogeographic histories is expected to contribute for understanding flora formation in East Asia and the Far East. 

Regional flora should never conclude in itself but be compared with those of neighboring regions toward a global view. In widespread species or species complexes, we often troubled when applying the results of evidential revisional works with limited geographical coverage of samplings to regional flora of unsampled areas. As Shrestha and Zhang (2014) demonstrated in the Chinese populations, Huperzia serrata sensu lato (Lycopodiaceae) is never a single species but an aggregation of multiple biological units, and it was supported by DNA content analyses using populations in Aichi Prefecture of Japan (Aman et al. 2011). Unfortunately, these findings were not reflected but recognized only Huperzia serrata s.l. in FernGreenList 1.0.1 (Ebihara et al. 2016) as we could not drew reliable synthesis of these results for applying Japanese populations. There are a number of native species or groups in Japan which are awaiting careful comparison with published results mostly generated in China and other Asian countries (e.g. Pteridium [Zhou et al. 2015], Hymenasplenium [Xu et al. 2018]) and Leptochilus [Zhang et al. 2019].

Contrasting to a consensus on familial classification of ferns and lycophytes almost achieved in PPGI (Pteridophyte Phylogeny Group 2016), generic classification proposed in the system requires a good deal of update and emendation based on newly generated evidences. For example, the 30-generic system of Thelypteridaceae adopted in the PPGI system seems hardly applicable to the species in Japan due to generation of a nothogenus x Chrinephrium (x C. insulare) and due to uncertain placement for the species formerly included in Parathelypteris (e.g. the material of “Parathelypteris nipponica” from China used in the phylogeny of He and Zhang (2012) and Almeida et al. (2016) is in question for the identification). It is important to publicize data on East Asian material with accurate identification working as bases for generic recircumscription in forthcoming “PPGII”.

Accumulation of data on ploidy and reproductive modes along with recognition of proper taxonomic units and their distribution data enables us to discuss about various characteristics of ferns and lycophytes as an organism group such as a proportion of polyploids and a proportion of apomict. In a country scale, combined analyses between the occurrence data provided by the present study and various kinds of geographic variables may yield further new findings but we here confine preliminary ones. We finally note an interesting finding which is not exposed in present maps: a hotspot of diploid progenitors which are probable palaeoendemics on the riverside in Amami-oshima Island, central Ryukyu. The hotspot is represented by at least four examples: 1) Polystichum obae (Dryopteridaceae), an endemic diploid species (Takamiya 1996) with a quite small population, and is sister to a clade consisting of tetraploid P. polyblepharon (Takamiya 1996) + diploid P. parvipinnulum (Tsai and Shieh 1985) in plastid phylogeny (Le Pechon et al. 2016); 2) Coniogramme gracilis (Pteridaceae), a diploid species (Kurita 1972) endemic to the island, and is presumably a progenitor of widespread tetraploid C. japonica (Kurita 1963); 3) Diplazium amamiana (Athyriaceae), a diploid species (Takamiya et al. 1999) endemic to the island, and sister to widespread apomictic species D. hachijoense in plastid phylogeny (Ebihara 2011, Wei et al. 2013); 4) Pteris oshimensis (Pteridaceae), a widespread species, and sexual diploid is only found in Amami Islands in contrast to widespread apomictic races (Nakato and Ebihara 2016). The pattern, sexual diploid confined to Amami Islands vs. widespread polyploids, shared by these four pairs strongly suggests that the island is a relic for ancestral diploids. However, this may be one of many outcomes from our present datasets of ferns and lycophytes in Japan. It is highly expected to be drawn evolutionary scenarios for formation of flora and diversity by accumulation and integration of huge but reliable information.

## References

```{r insert-refs, results = "asis"}
read_lines(here::here("manuscript/refs.md")) %>% cat(fill = FALSE, sep = "\n")
```

## Figures

Fig. 1. Maps of fern and lycophyte diversity in Japan on a 10 km^2^ grid. Taxa used to calculate diversity metrics include native species, subspecies, varieties; **a** and **f** also include hybrids, while **b**--**e** do not. **a** Richness of native fern and lycophyte taxa. **b** Phylogenetic diversity of fern and lycophyte taxa. **c** Phylogenetic diversity of fern taxa. **d** Richness of apomictic fern taxa. **e** Proportion of apomictic taxa out of all taxa. **f** Richness of threatened fern and lycophyte taxa included in the national red list of Japan. Note that the color gradients for **e** and **f** are on a log-scale.

## Electronic supplementary materials

Electronic supplementary material 1. A list of native fern and lycophyte taxa (species, subspecies and varieties) in Japan accepted in this study. Taxon ID refers that in FernGreenList ver.1.0.1 (http://www.rdplants.org/gl/). Otherwise noted, rbcL GenBank accession numbers are those used in Ebihara et al. (2010). Information on reproductive modes and ploidy levels follow those in Ebihara et al. (2016, 2017), and only the records based on material collected in Japan are accepted. For reproductive mode, irregular meiosis is not considered, 0 = no information, 1 = sexual, 2 = apomictic and 3 = sexual + apomictic. 

Electronic supplementary material 2. A list of herbarium specimens of 720 native fern and lycophyte taxa in Japan which the distribution maps in ca. 10-km grids in Ebihara et al. (2016, 2017) based on. Small number of corrections on identification and location after the publication were applied, but findings (e.g. new species and new records) after the publication are not reflected. Taxon ID refers that in FernGreenList ver.1.0.1 (http://www.rdplants.org/gl/). Secondary grid code of Japan (http://www.stat.go.jp/english/data/mesh/05.html) and coordinates of their center points are listed. Herbarium acronyms follow Index Herbariorum (http://sweetgum.nybg.org/science/ih/) but the following abbreviations are used for those not listed in Index Herbariorum; Gifu: Gifu Prefectural Museum; Gunma: Gunma Museum of Natural History; Iwate: Iwate Prefectural Museum; Kobe: Kobe University; Kurashiki: Kurashiki Museum of Natural History; Miyajima: Miyajima Natural Botanical Garden, Hiroshima University, Niigata: Niigata University; Sendai City Wild Plants Garden; Tochigi: Tochigi Prefectural Museum; Yamaguchi: Yamaguchi Prefectural Museum; Yokohama; Yokohama City Children’s Botanical Garden; Yokosuka: Yokosuka City Museum. All the listed specimens were seen by the authors except for a part of Isoetes specimens which are cited from Takamiya et al. (1997).  

Electronic supplementary material 3. JpFernKeys ver. 1.0: An interactive key file for identification of all the native fern and lycophyte taxa (species, subspecies and varieties) in Japan plus a few common naturalized species for the software Lucid v3.3 (free edition) available at http://www.lucidcentral.com. The data are excerpts from matrices of morphological characters included in Ebihara (2016, 2017). The measurements were based on each ten fertile herbarium specimens collected in Japan which were selected randomly with a few exceptions for the species bearing large fronds (e.g. tree ferns). For some rare species with less than ten specimens available, those collected outside Japan were used. 
